recipes/shared-memory-dict/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry
```

Need to include `python` in `run`

```suggestion
    - poetry
  run:
    - python
```